### ISSUE TITLE
Exand Zips, potentially rename project

### ISSUE THREAD

**Issue by realJoshByrnes** [CymatiCorp/chrome-for-mirc#2]
I have submitted a pull request with expanded zip files.
I'd also urge you to consider renaming this project.

Whilst it absolutely contains **Chromium**, it does not contain **Chrome**, which is a proprietary browser by Alphabet's Google.
Other than the obviously misleading name, I'd expect that there could be potential legal issues with using the Chrome name without permission, although I am not a lawyer.

**Comment by warrenarea** [comment]
well, i renamed it to browser-for-mirc

in your pull request, do you think you could put all
those files, into folders, so its not all in the root directory.

as you can probably tell by now, not very experienced with git.

i really just wanted to make sure the DLL's were available for everyone
to make use of... and the source code is just there if anyone can make 
use of it.

this actually requires a huge overhaul anyways... i'm not even sure if it's
worth it.... i ended up using DCX.dll for its web browser... it seemed to be
more stable, and had a lot less javascript conflicts.  This one is a bit glitchy,
not even sure where to start, if it'd be worth my time, or if anyone else shares
interest. 

last time i expressed my interest to the writers of DCX, they kind of just 
discouraged me and told me off saying developing anything for mIRC is a
dead end and a waste of time.   so can't say that was very inspiring.

**Comment by warrenarea** [comment]
javascript seems to be really screwed up with it, so not much functionality with this.
probably has a lot of permission issues.

**Comment by realJoshByrnes** [comment]
> well, i renamed it to browser-for-mirc


Good idea, simple name, although I would have personally used Chromium or CEF (Chromium Embedded Framework)

> in your pull request, do you think you could put all
> those files, into folders, so its not all in the root directory.


Most of the items are in src/bin folders
 
> as you can probably tell by now, not very experienced with git.


Can't tell, but it takes time. The good news is that once you get the hang of the basics, it pretty much all fits in to place. I can highly recommend Visual Studio code for managing git projects. (there's a little share button)

> i really just wanted to make sure the DLL's were available for everyone
> to make use of... and the source code is just there if anyone can make
> use of it.


Good stuff!


> this actually requires a huge overhaul anyways... i'm not even sure if it's
> worth it.... i ended up using DCX.dll for its web browser... it seemed to be
> more stable, and had a lot less javascript conflicts. This one is a bit glitchy,
> not even sure where to start, if it'd be worth my time, or if anyone else shares
> interest.


I'm not sure what you mean by Glitchy - I had a lot of success browsing to a local page and using Javascript. I was mixing JS and mSL to do stuff (mostly JS).
As per my reply to you on the mIRC forums, I'd probably say that it's a really good idea to use a recent version of CEF from https://github.com/chromiumembedded/cef or one of the related external projects.


> last time i expressed my interest to the writers of DCX, they kind of just
> discouraged me and told me off saying developing anything for mIRC is a
> dead end and a waste of time. so can't say that was very inspiring.


That's pretty normal for devs that don't use something anymore. The value YOU place on the project is not always the same as others, particularly if they have no use for it anymore.

**Comment by 3ncriptado** [comment]
Greetings, I would like to have help to implement this in my Ircap, when trying to Open the Alias TEST set in the MRC that is in BIN it tells me the following error: * $ dll: unable to open 'C: \ Users \ maico \ Desktop \ bot irc \ Sscript \ addons \ mChef.dll '(line 31, mChef.mrc)

The file is in the specified path but the MRC is unable to get the reading, any ideas?




In spanish Languaje:
Saludos, Quisiera tener ayuda para implementar esto en mi Ircap,  al intentar  Abrir el  Alias TEST puesto en el MRC  que hay en BIN me dice el siguiente error: * $dll: unable to open 'C:\Users\maico\Desktop\bot irc\Sscript\addons\mChef.dll' (line 31, mChef.mrc)
El archivo se encuentra en la ruta especificada pero le es imposible al MRC  obtener la lectura, alguna idea?

**Comment by realJoshByrnes** [comment]
@maicoldlx Please start a new issue for new issues.

The first two things that come to mind are target incompatibilities (is mIRC in 32bit mode, what platform did you build the DLL for? etc)

The second thing is do you have a virus scanner? Is it locking the file due to a false positive?

Lastly, are there any missing dependancies?

You haven't described how you've run it, or anything about your system, these could be helpful.

**Comment by 3ncriptado** [comment]
Oh sorry

I didn't even know that I had to compile the DLL or build it.

I don't have much knowledge about DLL, everything I have done for Irc Ircap, I have done without DLL.

If they could help me it was a success, I think that not having an Element like Chromiun in IRCAP, is what stops me from doing other great projects.

Right now I have two projects in Development.
1: A mobile App with Remote Connection to Ircap to have all its functions
2: A translator with the google API (already functional but lacks some improvements)

Spanish:

OH, perdona, 

yo ni sabía que tenia que compilar la DLL  o construirla. 

No tengo mucho conocimiento sobre DLL, todo lo que hecho para Irc Ircap, lo he hecho sin DLL. 

Si pudieran ayudarme fuera un exito, yo creo que el no tener un Elemento COmo Chromiun en IRCAP, es lo que me detiene de hacer otros grandes proyectos.

Ahora mismo llevo en Desarrollo dos proyectos.
1: Una App movil con Conexion Remota a Ircap para tener todas sus funciones
2: Un traductor con la API de google (ya funcional pero le faltan algunas mejoras)
